smithy-model/src/main/resources/software/amazon/smithy/model/loader/prelude-traits.smithy

```diff
@@ -156,7 +156,7 @@ structure xmlFlattened {}
 /// used in the model.
 @trait
 @tags(["diff.error.const"])
-@pattern("^[a-zA-Z_][a-zA-Z_0-9-]*$")
+@pattern("^[a-zA-Z_][a-zA-Z_0-9-]*(:[a-zA-Z_][a-zA-Z_0-9-]*)?$")
```

Comment:
This pattern update should be reflected in the [`xmlName` trait spec's ABNF](https://awslabs.github.io/smithy/spec/xml.html#xmlname-trait).